app/templates/events/view/tickets/access.hbs

```diff
@@ -0,0 +1,74 @@
+{{#if device.isMobile}}
+  <br><br>
+{{/if}}
+<div class="ui {{if device.isMobile 'centered'}} grid">
+  <div class="{{if device.isMobile '' 'row'}}">
+    <h1 class="ui header {{if device.isMobile 'sixteen' 'six'}} wide column">{{t 'Access Codes'}}</h1>
+    <div class="{{if device.isMobile 'sixteen' 'ten'}} wide column right aligned"><button class="ui button primary">Create an access code</button></div>
+  </div>
+  <div class="{{if device.isMobile '' 'row'}}">
+    {{#if device.isMobile}}
+      <br>
+    {{/if}}
+    <div class="ui buttons">
+      <button class="ui button">{{t 'All'}}</button>
+      <button class="ui button">{{t 'Active'}}</button>
+      <button class="ui button">{{t 'Inactive'}}</button>
+    </div>
+    {{#if device.isMobile}}
+      <br><br>
+    {{/if}}
+    <div class="{{if device.isTablet '' 'right floated ten wide column right aligned'}}">
+      <div class="ui search">
+        <div class="ui icon input">
+          <input class="prompt" type="text" placeholder="Search">
+          <i class="search icon"></i>
+        </div>
+        <div class="results"></div>
+        &nbsp;&nbsp;
+        {{#if device.isMobile}}
```

Comment:
For now, I don't think there is a need of so much styling please keep the general look and feel similar across the tabs.